buildpipeline/perf-pipeline.groovy

```diff
@@ -19,25 +32,32 @@ def windowsBuild(String arch, String config, String pgo, boolean isBaseline) {
     }
 
     bat "set __TestIntermediateDir=int&&.\\build.cmd -${config} -${arch} -skipbuildpackages ${pgoBuildFlag}"
+    bat "tests\\runtest.cmd ${config} ${arch} GenerateLayoutOnly"
+    bat "rd /s /q bin\\obj"
 
     // Stash build artifacts. Stash tests in an additional stash to be used by Linux test runs
-    stash name: "nt-${arch}-${pgo}${baselineString}-build-artifacts", includes: 'bin/**'
+    stash name: "nt-${arch}-${pgo}${baselineString}-build-artifacts", includes: 'bin/'
```

Comment:
Why did we get rid of the recursive stash of the bin directory here? Was it no longer needed as we had no other directories under bin, or is this part of the size reduction?

Reply:
That's a mistake